MoreLinq.Test/BreakingListExtensions.cs

```diff
@@ -0,0 +1,19 @@
+namespace MoreLinq.Test
+{
+	using System;
+	using System.Collections.Generic;
+	using System.Linq;
+	using System.Text;
+	using System.Threading.Tasks;
+
+	static class BreakingListExtensions
```

Comment:
This can be named `BreakingList` and folded into the same file as `BreakingList.cs`, which would simplify things a lot.

Reply:
Can't you only write extension methods in non-generic static classes?

Reply:
I meant put both classes in the same file. See `TestSequence.cs`, for example.